utils.py

```diff
@@ -188,17 +188,48 @@ def validate_cookies_del(flows: list[dict]) -> None:
                 f"when deleting flows. flow: {flow}"
             )
 
-
-def flows_to_log(logger_fun: Callable, message: str, flow_dict: dict[str, list]):
-    """Log flows, maximun flows in a log is 200"""
-    length_msg = f"total_length: {len(flow_dict['flows'])}, "
+def flows_to_log(
+    logger_fun: Callable,
+    message: str,
+    switches: list[str],
+    flows_dict: Union[
+        dict[str, dict[str, list]], # by_switch=True
+        dict[str, list]             # by_switch=False
+    ],
+    by_switch=False
+):
+    """Log the information of installing or deleting flows. The logs will
+     show the quantity of flows being modified in each switch or all
+     existent switches.
+     If flows are sent by switch, each switch will log the flows modified."""
+    log_str = "Batched of flows received: "
+    count_flows = 0
+    for switch in switches:
+        if not by_switch:
+            flows_n = len(flows_dict['flows'])
+            log_str = log_str + f" switches:{switches}, flows_by_switch:{flows_n}, "
+            count_flows = flows_n*len(switches)
+            _flows_to_log(logger_fun, message, switches, flows_dict['flows'])
+            break
+        
+        flows_n = len(flows_dict[switch]['flows'])
+        log_str = log_str + f"{{switch: {switch}, flows_lenght: {flows_n}}}, "
```

Comment:
```suggestion
        log_str = log_str + f"{{switch: {switch}, flows_length: {flows_n}}}, "
```

typo on log str

Reply:
Fixed it but added a "q", my bad. Next commit.